plugins/outputs/sensu/README.md

```diff
@@ -0,0 +1,96 @@
+# HTTP Output Plugin
+
+This plugin writes metrics events to [Sensu Go](https://sensu.io)  via its 
+HTTP events API.
+
+### Configuration:
+
+```toml
+  ## BACKEND API URL is the Sensu Backend API root URL to send metrics to 
```

Comment:
```suggestion
```toml
[[outputs.sensu-go]]
  ## BACKEND API URL is the Sensu Backend API root URL to send metrics to 
```